.github/workflows/build.yml

```diff
@@ -86,19 +86,19 @@ jobs:
 
       - name: Run Bicep E2E Tests
         if: ${{ matrix.rid != 'linux-musl-x64' }}
-        run: npm ci && npm run compile && npm test
+        run: npm ci && npm test
```

Comment:
>test [](http://example.com/codeflow?start=27&length=4)

Can you also update the official build?

Reply:
Are we running CLI E2E tests in official build? I could not find the task for it.

Reply:
Oh. Good pt. I assumed we were, but it's possible we missed it. We will need to add them in there if they are missing, though.

Reply:
But that could be a separate PR.